common/source/docs/common-pixhawk-camera-trigger-setup.rst

```diff
@@ -19,7 +19,7 @@ Items Required
 ==============
 
 -  Sony A6000 (or other camera with wire shutter release)
--  ArduPilot compatible autopilot
+-  ArduPilot compatible autopilot with a free GPIO pin. See :ref:`common-gpios` for more information
```

Comment:
I think if we are talking about the autopilot controlling a GPIO pin we should actually stick with the word "Relay".